.gitignore

```diff
@@ -49,6 +49,8 @@ src/*
 
 .DS_store
 
+nul
```

Comment:
@nociale 
Whats a  `nul` file ?

Reply:
When getting the pdf of the joss paper from my local pc I was getting this "nul" file which is just an empty file. I could not even delete it, so I told git to ignore it.